admin/class-admin-init.php

```diff
@@ -103,9 +103,8 @@ private function on_wpseo_admin_page() {
 	 * Determine whether we should load the meta box class and if so, load it.
 	 */
 	private function load_meta_boxes() {
-
 		$is_editor      = WPSEO_Metabox::is_post_overview( $this->pagenow ) || WPSEO_Metabox::is_post_edit( $this->pagenow );
-		$is_inline_save = filter_input( INPUT_POST, 'action' ) === 'inline-save';
+		$is_inline_save = check_ajax_referer( 'inlineeditnonce', '_inline_edit', false ) && isset( $_POST['action'] ) && $_POST['action'] === 'inline-save';
```

Comment:
Can we not have this as an inline check, to improve readability?

Also, even though it's just a equality check, best if we still did the sanitize_text_field/wp_unslash thing imo

Reply:
Also, the `improve readability` part @KiOui ? :D